contents/blog/posthog-vs-kubit.md

```diff
@@ -0,0 +1,412 @@
+            <td className="text-center"><span className="text-green text-lg">✔</span></td>
+        </tr>
+        <tr>
+            <td><strong>Data anonymization</strong></td>
+            <td className="text-center"><span className="text-red text-lg">✖</span></td>
+            <td className="text-center"><span className="text-green text-lg">✔</span></td>
+        </tr>
+        <tr>
+            <td><strong>Cookie-less tracking option</strong></td>
+            <td className="text-center"><span className="text-red text-lg">✖</span></td>
+            <td className="text-center"><span className="text-green text-lg">✔</span></td>
+        </tr>
+        <tr>
+            <td><strong>SOC 2 certified</strong></td>
+            <td className="text-center"><span className="text-green text-lg">✔</span></td>
+            <td className="text-center"><span className="text-green text-lg">✔</span></td>
+        </tr>
+        <tr>
+            <td><strong>SAML/SSO available</strong></td>
+            <td className="text-center"><span className="text-green text-lg">✔</span></td>
+            <td className="text-center"><span className="text-green text-lg">✔</span></td>
+        </tr>
+        <tr>
+            <td><strong>2FA available</strong></td>
+            <td className="text-center"><span className="text-red text-lg">✖</span></td>
+            <td className="text-center"><span className="text-green text-lg">✔</span></td>
+        </tr>
+    </tbody>
+</table>
+</div>
+
+> ### To cookie, or not to cookie?
+>
+> To ensure compliance with local privacy regulations, you may need to ask users for their permission to track them with cookies. PostHog has [tools to help you do that](/tutorials/react-cookie-banner), but if you’d rather not use cookies to track users then [you can do that too](/tutorials/cookieless-tracking). 
+
+## Frequently asked questions
+
+### How much should I expect to pay?
+
+With Kubit, it’s impossible to estimate costs as pricing is not transparent and is tailored to each customer. At PostHog, [we strongly disagree with this approach](/blog/transparent-enterprise-pricing) and publish our pricing publicly.
+
+For PostHog, we have [a guide for estimating your usage](/docs/billing/estimating-usage-costs), and [a pricing calculator](/pricing) to help — but with a generous free tier and no up-front fees, it’s often easiest to just get started and see how much data you send through that way. 
+
+### Is there a free trial?
+
+With PostHog, you don’t need a free trial — it’s free to get started, with a generous monthly allowance of events, replays, and API calls. If you’re within this allowance, PostHog is free to use forever. 
+
+Kubit does offer a free trial, but you’ll need to submit your company information and wait for Kubit to respond with an invite to do so. We tried this ourselves, and have been waiting a week for an invite or someone to reach out. 
```

Comment:
```suggestion
Kubit does offer a free trial, but you’ll need to submit your company information and wait for Kubit to respond with an invite to do so. 
```

See earlier comments

Reply:
Will commit this, but see response above